### DISCUSSION TITLE
Azure Data Studio Icon Please

### DISCUSSION THREAD

**Discussion by neeku** [elrumo/macOS_Big_Sur_icons_replacements#1472] [Icon request]
Thanks to this amazing website, I replaced several blue-colored icons, but I can't find an alternative for the Azure Data Studio. Would a kind soul be able to create one?

**Comment by itjustcrashed** [comment]
ima need some more info on dat. like, imagine and icon for it and tell me what colors you want + what you like / dislike about it.
